### ISSUE TITLE
looking forward the pytorch version

### ISSUE THREAD

**Issue by sxzy** [bryanyzhu/Hidden-Two-Stream#36]
hello when will have the pytorch release

**Comment by bryanyzhu** [comment]
Hi, thank you for your interest in this work. But unfortunately, I am working on several projects right now, and has no time to release the PyTorch code. 

If you want to do it yourself, it should be simple, just by combining these two repos together, which is what I did. 

https://github.com/ClementPinard/FlowNetPytorch
https://github.com/bryanyzhu/two-stream-pytorch

All things you need is basically there, like the dataloader, loss function, network architecture, etc. Sorry for the inconvenience.
